Common/OJS.Common/Extensions/RuleBuilderExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace OJS.Common.Extensions;
+
+using FluentValidation;
+using System;
+
+public static class RuleBuilderExtensions
+{
+    public static IRuleBuilderOptions<T, string?> MustBeValidEnum<T, TEnum>(this IRuleBuilder<T, string?> ruleBuilder)
+        where TEnum : Enum
+        => ruleBuilder.Must(BeValidEnum<TEnum>).WithMessage($"The value is not of type {typeof(TEnum)}");
```

Comment:
I think the exception message will not be very descriptive.  what will `typeof(TEnum)` be in string representation?

Reply:
Fixed.